.github/workflows/ci.yml

```
@@ -79,11 +79,12 @@ jobs:
             type=raw,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/v') }}
 
       - name: Build and load docker image
```

Let's add the buildx setup action here:
```suggestion

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2

      - name: Build and load docker image
```